packages/api/src/user.js

```diff
@@ -183,6 +183,24 @@ export async function userTokensPost (request, env) {
   return new JSONResponse(res.createAuthToken, { status: 201 })
 }
 
+/**
+ * Retrieve user storage data.
+ *
+ * @param {AuthenticatedRequest} request
+ * @param {import('./env').Env} env
+ */
+export async function userStorageGet (request, env) {
+  const res = await env.db.query(gql`
+    query findStorageByUser($user: ID!) {
```

Comment:
Just use the existing `findUserByID` that fauna generates?

Reply:
I looked into the generated functions yesterday and I couldn't find it lol 🤦🏼‍♂️  But now I see that the place to find them is in the docs tab in graphql section. So yes, of course let's just use that one